### PR TITLE
Updated collator setup guide

### PR DESCRIPTION
**Pull request by PoisonPhang**
## Issue
N/A

## Description
Updated the collator setup guide using [this document](https://docs.google.com/document/d/1Ub76PGcS0dXWa0G-q064l9EKar15WPweLQHlAQiozjA/edit#)

## Checklist

- ~I have updated the cargo docs to reflect changes made by this PR~
- [X] I have updated the `book/` to reflect changes made by this PR

### CODE REVIEW

File: book/src/developer-guides/collator-set-up-guide.md [hunk 1]
```diff
@@ -65,51 +58,52 @@ export SKIP_WASM_BUILD=1 && \
...
 
 ```sh
 sudo systemctl status collator.service
 ```
 
-5. Check logs output
-
+Check logs output
 ```sh
 journalctl -f
```
Comment (dzmitry-lahoda) [inline book/src/developer-guides/collator-set-up-guide.md:102]: seems it should higher in document?

File: book/src/developer-guides/collator-set-up-guide.md [hunk 2]
```diff
@@ -1,27 +1,19 @@
-![picasso-bannar](./picasso-bannar.png)
+# Collator Setup Guide
 
-# COLLATOR SET UP GUIDE
-
-In this document we will cover how to set up a collator with Composable 
-Finance. There are several ways to do that.
+In this document we will cover how to set up a collator with Composable Finance.
```
Comment (dzmitry-lahoda) [inline book/src/developer-guides/collator-set-up-guide.md:3]: Currently this is Picasso, please tell it in top of guide

File: book/src/developer-guides/collator-set-up-guide.md [hunk 3]
```diff
@@ -37,16 +29,17 @@ rustup default $RUST_C && \
...
 
-### Get project and build node
+Get project and build node
 
 ```sh
 git clone --depth 1 --branch v2.1.6 https://github.com/ComposableFi/composable.git && \
 cd composable && \
 export SKIP_WASM_BUILD=1 && \
-cargo build --release
+cargo build --release
```
Comment (andor0) [inline book/src/developer-guides/collator-set-up-guide.md:38]: trailing space

File: book/src/developer-guides/collator-set-up-guide.md [hunk 4]
```diff
@@ -65,51 +58,52 @@ export SKIP_WASM_BUILD=1 && \
...
 
-2. Create collator.service file and put following content into it. Save it with Ctrl+O
+Create `collator.service` file and put following content into it. Save it with `Ctrl+O`
 
 ```sh
 sudo nano /etc/systemd/system/collator.service
 ```
 
-3. Enable and Start service
+Enable and Start service
```
Comment (andor0) [inline book/src/developer-guides/collator-set-up-guide.md:87]:
```suggestion
Enable start at boot and start service
```

File: book/src/developer-guides/collator-set-up-guide.md [hunk 5]
```diff
@@ -209,10 +203,49 @@ services:
       --port 30333
       --unsafe-ws-external
       --unsafe-rpc-external
-      --listen-addr=/ip4/0.0.0.0/tcp/30334
+      --listen-addr=/ip4/0.0.0.0/tcp/30334k
```
Comment (andor0) [inline book/src/developer-guides/collator-set-up-guide.md:206]:
```suggestion
      --listen-addr=/ip4/0.0.0.0/tcp/30334
```

File: book/src/developer-guides/collator-set-up-guide.md (hunk 3 above)
Comment (andor0) [inline book/src/developer-guides/collator-set-up-guide.md:38]:
```suggestion
cargo build --release
```

File: book/src/developer-guides/collator-set-up-guide.md [hunk 6]
```diff
@@ -209,10 +203,49 @@ services:
...
       --execution wasm
       --
       --execution=wasm 
       --listen-addr=/ip4/0.0.0.0/tcp/30333
 ```
 
+### How to start the Composable node with Granfana
+
+```sh
+cd docker-compose-collator/
```
Comment (andor0) [inline book/src/developer-guides/collator-set-up-guide.md:216]: where can find this directory?

File: book/src/developer-guides/collator-set-up-guide.md [hunk 7]
```diff
@@ -183,7 +177,7 @@ docker container run hello-world
 
 ### docker-compose
 
-Save the following content to docker-compose.yml and run docker-compose up -d
+Save the following content to `docker-compose.yml` and run `docker-compose up -d`
 
 ```yml
```
Comment (dzmitry-lahoda) [inline book/src/developer-guides/collator-set-up-guide.md:182]: May be move it into separate file and just reference?

File: book/src/developer-guides/collator-set-up-guide.md [hunk 8]
```diff
@@ -209,10 +203,49 @@ services:
...
+### How to start the Composable node with Granfana
+
+```sh
+cd docker-compose-collator/
+
+docker-compose up 
+
+```
+
+Prometheus should be available on `127.0.0.1:9090`
```
Comment (dzmitry-lahoda) [inline book/src/developer-guides/collator-set-up-guide.md:222]: may be make it a link?

File: book/src/developer-guides/collator-set-up-guide.md [hunk 9]
```diff
@@ -209,10 +203,49 @@ services:
...
+
+Cadvisor `127.0.0.1:8080`
+
+Grafana `127.0.0.1:3000`
+
+Default login/password is `admin`/`admin123`
+
+To make grafana work, prometheus needs to be added as data source 
+
+![grafana-1](./grafana-1.png)
```
Comment (dzmitry-lahoda) [inline book/src/developer-guides/collator-set-up-guide.md:232]: may be LFS? Or we end up not being monorepo soon... i guess

**Comment by RAVIKUMAR96** [comment]
Hey @dzmitry-lahoda is this guide need to fixed/updated by SRE if that so , I can open up a ticket on SRE board ,please let me know

**Pull request closed without merging**